src/snowflake/cli/_plugins/dbt/constants.py

```diff
@@ -0,0 +1,20 @@
+DBT_COMMANDS = [
+    "build",
+    "clean",
+    "clone",
+    "compile",
+    "debug",
+    "deps",
+    "docs",
+    "init",
+    "list",
+    "parse",
+    "retry",
+    "run",
+    "run",
+    "seed",
+    "show",
+    "snapshot",
+    "source",
+    "test",
+]
```

Comment:
Should we consider some feature flag for enabling new commands. Something like `SNOWFLAKE_FEATURES_ENABLE_DBT_COMMANDS=foo,bar,baz`?